CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Keycloak Community
+
+The Keycloak Node.js adapter is an Open Source adapter solution to protect modern Node.js Applications and Services.
+
+## Contributing to the Node.js adapter
+
+The Node.js adapter is an Open Source community-driven project and we welcome contributions as well as feedback from the community.
+
+We do have a few guidelines in place to help you be successful with your contribution to the Node.js adapter.
+
+Here's a quick checklist for a good PR, more details below:
+
+1. [Keycloak Dev Mailing List](https://lists.jboss.org/mailman/listinfo/keycloak-dev)
```

Comment:
@vmuzikar before implement anything new or submit a quickstart. It has to be discussed on the dev mailing list first, otherwise people may waste their time with something we don't need.

This checklist is pretty much the same from keycloak/keycloak, keycloak/keycloak-gatekeeper and keycloak/keycloak-nodejs-connect. We are trying to keep it pretty much standard.

Reply:
Ok, fair enough. Take this as resolved. ;)